examples/hello-node-express-ts/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "hello-node-express-ts",
+  "version": "0.0.1",
+  "description": "Honeycomb OpenTelemetry Distro for JavaScript example",
+  "author": "Honeycomb <support@honeycomb.io> (https://www.honeycomb.io/)",
+  "license": "Apache-2.0",
+  "main": "dist/src/index.js",
+  "scripts": {
+    "build": "tsc --build",
+    "setup": "npm install && cd ../.. && npm install && npm run build",
+    "start": "ts-node -r ./instrumentation.ts index.ts",
+    "start-js": "node -r ./dist/instrumentation.js ./dist/index.js"
+  },
+  "dependencies": {
+    "@honeycombio/opentelemetry-node": "file:../../dist/src",
+    "@opentelemetry/auto-instrumentations-node": "^0.34.0",
+    "@types/express": "^4.17.14",
+    "@types/node": "^18.11.9",
+    "express": "^4.18.2",
+    "ts-node": "^10.9.1"
```

Comment:
just realized this should prolly go into devDependencies

Reply:
https://github.com/honeycombio/honeycomb-opentelemetry-node/pull/112/commits/016ba30c1033402af0721359aa670ae36de3c107